src/main/java/org/spongepowered/api/Game.java

```diff
@@ -22,7 +22,6 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-
```

Comment:
I don't mind a new line after the header useful or needed at best. We won't be doing that.